tools/fpgainfo/errors.c

```diff
@@ -316,6 +337,29 @@ fpga_result errors_filter(fpga_properties *filter, int argc, char *argv[])
 	return res;
 }
 
+static fpga_result get_port_error_revision(fpga_token token, uint64_t *value)
+{
+	fpga_result res = FPGA_OK;
+	fpga_object fpga_object;
+
+	res = fpgaTokenGetObject(token,"error*/revision" , &fpga_object, FPGA_OBJECT_GLOB);
+	if (res != FPGA_OK) {
+		OPAE_MSG("Failed to get token Object");
+		return res;
+	}
+
+	res = fpgaObjectRead64(fpga_object, value, 0);
+	if (res != FPGA_OK) {
+		OPAE_MSG("Failed to Read object ");
+	}
+
+	res = fpgaDestroyObject(&fpga_object);
```

Comment:
If line 351 fails, we lose the error code here.

Reply:
I will fix that cut n paste error.